src/api/api.js

```diff
@@ -0,0 +1,65 @@
+const fetch = require('node-fetch');
+const APIKey = process.env.VUE_APP_API_KEY;
+
+const getServiceResponse = async (city) => {
```

Comment:
Oh, God!

`(argument) => {}` makes me crazy. I don't understand the purpose of `(` and `)` here. Everything should have its purpose